backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/spi/impl/InOutProducerFromShipmentScheduleWithHU.java

```diff
@@ -327,7 +327,8 @@ private I_M_InOut createShipmentHeader(final ShipmentScheduleWithHU candidate, f
 		final I_M_InOut shipment = InterfaceWrapperHelper.newInstance(I_M_InOut.class, processorCtx);
 		shipment.setAD_Org_ID(shipmentSchedule.getAD_Org_ID());
 
-		//
+		shipment.setM_SectionCode_ID(candidate.getM_ShipmentSchedule().getM_SectionCode_ID());
```

Comment:
At yesterday's scrum we talked about the possibility that a BPartner's section code might change.
That means that you can have in two shipment-schedules the same `C_BPartner_ID` but different `M_SectionCode_ID`s.
Those scheds need to go into different shipments, and therefore the `M_SectionCode_ID` needs to be made a part of the `HeaderAggregationKey`